samples/tab-channel-group/mvc-csharp/teamsapp.local.yml

```diff
@@ -4,7 +4,7 @@
 version: v1.2
 
 additionalMetadata:
-  sampleTag: Microsoft-Teams-Samples:tab-channel-group-csharp
+  sampleTag: tab-channel-group-mvc-csharp
```

Comment:
Please DO NOT remove 'Microsoft-Teams-Samples:'!
The format is `<repo_name>:<sampe_name>`

Reply:
Fixed